src/style/_overrides.scss

```diff
@@ -384,23 +447,45 @@ $adminnav-border-bottom: 1px solid $gray-light;
   display: none;
 }
 
+// project-info
+md-virtual-repeat-container {
```

Comment:
1. so this needs to be more specific, as it will apply to all `md-virtual-repeat-container` we use, but we don't want that.
2.we should fill the entire accordion dropdown in the about page, not just 1/2.
3. I'm trying to move away from this file, and into more view/component specific files. This is super hard to read b/c it's so long.

I'd suggest removing this and adding an `_about.scss` file. You'll need to import it in `_fims.scss`.

Then you can do something like:

```
.about {
  project-info {
      md-virtual-repeat {
        ....
      }
  }
}
```